src/harness/reference_models/geo/census_tract.py

```diff
@@ -16,11 +16,14 @@
 """
 import json
 import os
-
+import zipfile
+import io
+import re
+import glob
+import shapefile
```

Comment:
All this would be part of the script itself.

Reply:
Fixed   as per suggestion.